netebpfext/net_ebpf_ext_bind.c

```diff
@@ -244,7 +244,7 @@ net_ebpf_ext_resource_allocation_classify(
 
 Exit:
     if (attached_client)
-        net_ebpf_extension_hook_client_leave_rundown(attached_client, EXECUTION_PASSIVE);
+        net_ebpf_extension_hook_client_leave_rundown(attached_client);
```

Comment:
Ideally we should call `net_ebpf_extension_hook_client_leave_rundown` only when "acquire rundown" was successful. Current code may work today, but if some other logic in future decides to re-initialize the rundown in a different flow, then this will cause problem.

Reply:
fixed.